### PR TITLE
Unit tests: remove unused imports

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
 Unit tests: remove unused imports, part 3

##### ISSUE TYPE

- Bugfix Pull Request

### CODE REVIEW

File: test/units/plugins/strategy/test_strategy_base.py [hunk 1]
```diff
@@ -418,7 +418,7 @@ def _has_dead_workers():
         # self.assertEqual(strategy_base._pending_results, 1)
 
         # queue_items.append(('bad'))
-        # self.assertRaises(AnsibleError, strategy_base._process_pending_results, iterator=mock_iterator)
+        # self.assertRaises(strategy_base._process_pending_results, iterator=mock_iterator)
```
Comment (felixfontein) [inline test/units/plugins/strategy/test_strategy_base.py]: I don't think this line should be modified.

Comment (Andersson007) [reply test/units/plugins/strategy/test_strategy_base.py]: @felixfontein , good job! bash automation affected.. ) I should have checked in detail. Thank you!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
Thanks for cleaning the tests up! :)

**Comment by Andersson007** [comment]
@felixfontein , thanks for looking at this

**Pull request merged**